### DISCUSSION TITLE
"--use-default" unexpected behavior: default values are missing. Bug or feature?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [koxudaxi/datamodel-code-generator#533] [Q&A] [answered]
Input:

```json
{
  "foo": 123456,
  "bar": "spameggs",
}
```
Current output (flags used:  `--snake-case-field --use-standard-collections --use-default`):
```python
class Model(BaseModel):
    foo: int
    bar: str
```

Expected output:
```python
class Model(BaseModel):
    foo: int = 123456
    bar: str = "spameggs"
```

**Real use case:** the field `foo` is ALWAYS equal to a some value. It can not be different. 

Related issue: https://github.com/samuelcolvin/pydantic/issues/1927, the problem is solved with the ` = Field(DEFAULT_VALUE, const=True)`

**Comment by koxudaxi** [comment, marked as answer]
I'm sorry for my too-late reply.
I guess you give the cli  JSON Data. 
The CLI will ignore the default value in JSON Data

**Reply by shaikmoeed** [reply]
@koxudaxi Is it possible to enable default value for JSON data as well? I was trying to add example value for each field by using `--field-include-all-keys` or `--use-default`, is there any possibility or flag available to do this from JSON data or flag to enable for JSON data too?

**Reply by NeroMorto** [reply]
@shaikmoeed Hi! Have you found a way to solve your problem?
